src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final int CAPACITY = 100;
+
     public static void main(String[] args) {
+        printWelcomeMessage();
+
+        Task[] tasks = new Task[CAPACITY];
+        int counter = 0;
+
+        while(true) {
+            String line;
+            Scanner input = new Scanner(System.in);
```

Comment:
Maybe you could move the Scanner object out of the while loop as it is being reinitialised on every iteration of the loop